scripts/deploy.ts

```diff
@@ -1,9 +1,27 @@
 import hardhat from "hardhat";
```

Comment:
We really shouldn't call this `deploy.ts` if deploying is just one fifth of what it does. I suggest you leave `deploy.ts` as it was and copy this new `deploy.ts` into a separate file called `quick-script.ts` (something of a `hardhat run` counterpart to `quick-test.ts`)